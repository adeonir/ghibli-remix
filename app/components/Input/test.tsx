import { render, screen } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'

import { Input } from '.'

describe('<Input />', () => {
  it(`should render correctly`, () => {
    const { container } = render(<Input type="text" />, {
      wrapper: MemoryRouter,
    })

    expect(screen.getByRole('textbox')).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
