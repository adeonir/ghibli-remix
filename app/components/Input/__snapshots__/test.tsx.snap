// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render correctly 1`] = `
<div>
  <input
    class="w-full rounded-lg bg-slate-600 px-4 py-3 text-slate-100 placeholder:text-slate-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-teal-500"
    type="text"
  />
</div>
`;
